app/services/users.js

```diff
@@ -7,7 +7,49 @@ const bcrypt = require('bcrypt'),
 
 const saltRounds = parseInt(config.common.bcrypt.saltRounds) || 10;
 
+const checkAlphanumeric = (string) => {
```

Comment:
I guess you moved all these logic to the service so you could reused it for the createAdmin logic? Thats one way of avoiding the code repetition, but now we have some responsibility issues because now the service has to check that all the params are present and valid, but that should be the controller's responsibility. So we should find a way to do all this in the controller and avoiding the code repetition.